common/lib/xmodule/xmodule/tests/helpers.py

```diff
@@ -74,10 +81,19 @@ def get_current_user(self):
         user = XBlockUser()
         if self.user.is_authenticated:
```

Comment:
We're not mocking the user anymore, so this is going to throw an `AttributeError` if the user is `None`.

I see that we don't want to mock the user because of the test in `common/lib/xmodule/xmodule/tests/test_lti_unit.py`. Therefore, let's add a safety check (`is not None`), to avoid throwing exceptions for default values of the class attributes.